api/pom.xml

```diff
@@ -30,7 +30,7 @@
 
     <groupId>jakarta.interceptor</groupId>
     <artifactId>jakarta.interceptor-api</artifactId>
-    <version>2.0.1-SNAPSHOT</version>
+    <version>2.1.0-SNAPSHOT</version>
```

Comment:
Should this be 2.1-SNAPSHOT?

Reply:
Not at the API level. The spec is 2.1, but the API jar is versioned down to the service release. This pom does not use the project root pom as its parent.

Reply:
I understand they are different. My point is that 2.1-SNAPSHOT is much simplier than 2.1.0-SNAPSHOT.

Reply:
I can't do a 2.1.0 release without using 2.1.0-SNAPSHOT as the dev version using the maven release plugin as far as I know. This is the procedure used by other projects, CDI, BV, ...

Reply:
ok. I've approved this PR.

Reply:
Can you merge it? I don't seem to have access.